var/spack/repos/builtin/packages/neurodamus-model/package.py

```diff
@@ -36,6 +36,17 @@ class NeurodamusModel(SimModel):
     variant('synapsetool', default=True,  description="Enable SynapseTool reader (for edges)")
     variant('mvdtool',     default=True,  description="Enable MVDTool reader (for nodes)")
     variant('common_mods', default='default', description="Source of common mods. '': no change, other string: alternate path")
+    variant('ngv',         default=False, description="Include NGV mod files")
```

Comment:
@ferdonline : possible to add variant for other neurodamus-models like thalamus, hippocampus etc?

Reply:
Hum... the idea is interesting (of an unique neurodamus recipe)... However I don't think we will concretize in this way, we probably are gonna build the several `nrnmechlibs` and let the user decide at runtime what to use (?)

Reply:
Oops, sorry! - my comment was missing a word!  What I wanted to say is that - possible to add variant `conflict` for other ...? i.e. `ngv` is only built with neocortex. And hence I was thinking about the conflict.

Reply:
`neuro-glia-vasculature` connections can happen anywhere, which is why we thought to put them in common models. With `neurodamus-model` being a parent class, basically any region can add ngv models.